pkg/gzip/gzip_tinygo.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build tinygo
+// +build tinygo
+
+package gzip
+
+import (
+	"io"
+
+	"compress/gzip"
+)
+
+// Compress takes input from io.Reader and deflates it using pgzip
```

Comment:
I think it should be
compress
not
Compress
one is proper English, the other proper Go function comment? :-)